src/calculateRentalCost.js

```diff
@@ -29,6 +29,24 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let sum;
+  const dayilyPay = 40;
+  const discountSmall = 20;
+  const discountBig = 50;
+
+  if (days < 3) {
+    sum = days * dayilyPay;
+  };
```

Comment:
```suggestion
  if (days < 3) {
    return (days * dayilyPay);
  };
```

  wrap expressions in brackets